cartographer/mapping_2d/pose_graph.cc

```diff
@@ -321,6 +321,22 @@ void PoseGraph::HandleWorkQueue() {
           trimmer->Trim(&trimming_handle);
         }
 
+        {
+          auto GetTrimmerToRemove = [&]() {
```

Comment:
Can't you use [std::remove_if](http://en.cppreference.com/w/cpp/algorithm/remove) here?

Reply:
didn't know this. ok